scripts/cola_baseline.sh

```diff
@@ -0,0 +1,20 @@
+
+# random-elmo
+rm *.out
```

Comment:
?

Reply:
Sorry, this one was because I made too many .out files during debugging. This shouldn't be in the commited version.